frontend/src/components/RecipeList.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+
+function RecipeList() {
+  const [recipes, setRecipes] = useState();
+
+  useEffect(() => {
+    async function getRecipeList() {
+      try {
+        const response = await axios.get('http://localhost:8081/api/recipes');
+        setRecipes(response.data);
+      } catch (err) {
+        console.error('Error during api call:', err);
+      }
+    }
+    getRecipeList();
+  }, []);
+
+  return (
+    <ul className="list-group">
+      {recipes
+        ? recipes.map((p) => (
+            <Link to={`/recipes/${p.id}`} key={p.id}>
+              <li className="list-group-item list-group-item-action">
```

Comment:
Ezt meg kellene cserélni, mert ez így invalid HTML.
Az ul-nek csak li lehet a gyereke: https://html.spec.whatwg.org/multipage/grouping-content.html#the-ul-element
Az li-n belül kellene lennie a linknek.

Reply:
megcserélve